Integrations/integration-AzureCompute.yml

```diff
@@ -0,0 +1,1085 @@
+            'PowerState': 'VM running'
+        }
+
+        title = 'Virtual Machine "{}" Started'.format(vm_name)
+        human_readable = tableToMarkdown(title, vm)
+        demisto.results({
+            'Type': entryTypes['note'],
+            'Contents': response,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': human_readable,
+            'EntryContext': {'Azure.Compute(val.Name && val.Name === obj.Name)': vm}
+        })
+
+
+    def poweroff_vm(args):
+        # Retrieve relevant command arguments
+        resource_group = args.get('resource_group')
+        vm_name = args.get('virtual_machine_name')
+
+        # Construct endpoint URI suffix
+        url_endpoint = resource_group + '/providers/Microsoft.Compute/virtualMachines/'
+        url_endpoint += vm_name + '/powerOff?api-version=' + API_VERSION
+
+        # Call API
+        update_access_token()
+        response = http_request('POST', url_endpoint, codes={202})
+
+        return response
+
+
+    def poweroff_vm_command():
+        """
+        Power-off a specified Virtual Machine
+
+        demisto parameter: (string) resource_group
+            Resource Group to which the virtual machine belongs
+
+        demisto parameter: (string) virtual_machine_name
+            Name of the virtual machine to power-off
+
+        returns:
+            Virtual Machine Object
+        """
+        args = demisto.args()
+        vm_name = args.get('virtual_machine_name')
+        response = poweroff_vm(args)
+
+        vm = {
+            'Name': vm_name,
+            'ResourceGroup': args.get('resource_group'),
+            'PowerState': 'VM stopped'
+        }
+
+        title = 'Virtual Machine "{}" Powered Down'.format(vm_name)
+        human_readable = tableToMarkdown(title, vm)
+        demisto.results({
+            'Type': entryTypes['note'],
+            'Contents': response,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': human_readable,
+            'EntryContext': {'Azure.Compute(val.Name && val.Name === obj.Name)': vm}
+        })
+
+
+    '''COMMAND SWITCHBOARD'''
+
+    commands = {
+        'azure-vm-list-instances': list_vms_command,
+        'azure-vm-get-instance-details': get_vm_command,
+        'azure-vm-start-instance': start_vm_command,
+        'azure-vm-poweroff-instance': poweroff_vm_command,
+        'azure-vm-create-instance': create_vm_command,
+        'azure-vm-delete-instance': delete_vm_command,
+        'azure-list-resource-groups': list_resource_groups_command
+    }
+
+    '''EXECUTION'''
+
+    try:
+        if demisto.command() == 'test-module':
+            update_access_token()
+            demisto.results('ok')
+        elif demisto.command() in commands.keys():
+            commands[demisto.command()]()
+
+    except Exception as e:
+        LOG(e)
```

Comment:
no need for this

Reply:
You mean I shouldn't include a LOG statement before `return_error`? Or the fact that I'm LOGging an Exception object, and not a string?

Reply:
`return_error()` already calls `LOG`, so no need to do it in integration code